varia/sass/elements/_media.scss

```diff
@@ -5,7 +5,7 @@ figure {
 figcaption {
 	color: #{map-deep-get($config-global, "color", "foreground", "light")};
 	font-size: #{map-deep-get($config-global, "font", "size", "xs")};
-	line-height: #{map-deep-get($config-global, "font", "line-height", "xs")};
```

Comment:
I couldn't find the `line-height.xs` mapping anywhere, so I changed it to `--line-height-body`. Is it ok for this instance?

Reply:
As with the other instance of this, I think `--line-height-heading` is best for captions so they take up less space.

Reply:
Done at b185f0b55e9361c0bdb70e5f3dcdd91223bc4306